topology/probes/netns/netns.go

```diff
@@ -284,6 +307,7 @@ LOOP:
 
 		case err := <-u.watcher.Errors:
 			logging.GetLogger().Errorf("Error while watching network namespace: %s", err)
+		case <-ticker.C:
```

Comment:
What is this ticker used for ?

Reply:
the check that the state changed otherwise if there is no event on the other chans it will block